modules/apps/frontend-theme/frontend-theme-dialect/src/WEB-INF/frontend-token-definition.json

```diff
@@ -4751,33 +4750,8 @@
 							],
 							"name": "hotkeysLgLineHeight",
```

Comment:
Following the pattern in other components like Buttons, the individual token should be in the singular form. ie: hotkeyLgLineHeight